instat/dlgView.vb

```diff
@@ -120,7 +118,6 @@ Public Class dlgView
         ucrReceiverView.SetMeAsReceiver()
 
         clsOutputWindowFunction.SetPackageName("utils")
-        clsSeparateWindowFunction.AddParameter("title", Chr(34) & ucrSelectorForView.ucrAvailableDataFrames.cboAvailableDataFrames.SelectedItem & Chr(34), iPosition:=1)

```

Comment:
Okay I see. What do you mean by the same conditions? Do you mean the same function is run?
I have that issue on some climatic dialogs anyway!